### ISSUE TITLE
segmentation.ipynb 

### ISSUE THREAD

**Issue by leowhynot** [Belousov-Aleksandr/PyTorch#1]
Некорректно считается валидационный лосс в train

avg_loss_val += loss / len(data_val)

1. loss_val, а не loss 
2.  Предложил бы более точную формулу расчета скора :
```
    score_val = iou_pytorch(torch.sigmoid(Y_pred_val) > 0.5, Yv_batch).sum().item()
    avg_score_val += score_val
avg_score_val = avg_score_val / len(val)
```

**Comment by leowhynot** [comment]
с расчетом среднего лосса тоже не согласен,
`avg_loss += loss / len(data_tr)`
в цикле прохода по батчам делит средний лосс по батчу на количество батчей в датасете. 
Разумнее было бы сделать так:
```
    avg_loss += loss 
avg_loss = avg_loss / len(data_tr)
```

**Comment by Belousov-Aleksandr** [comment]
> с расчетом среднего лосса тоже не согласен, `avg_loss += loss / len(data_tr)` в цикле прохода по батчам делит средний лосс по батчу на количество батчей в датасете. Разумнее было бы сделать так:
> 
> ```
>     avg_loss += loss 
> avg_loss = avg_loss / len(data_tr)
> ```

Илья! Спасибо за целевой комментарий! ))
